pkgs/by-name/cf/cfitsio/package.nix

```
@@ -12,13 +12,13 @@
 
 stdenv.mkDerivation (finalAttrs: {
   pname = "cfitsio";
-  version = "4.6.2";
+  version = "4.6.3";
 
   src = fetchFromGitHub {
     owner = "HEASARC";
     repo = finalAttrs.pname;
     tag = "${finalAttrs.pname}-${finalAttrs.version}";
```

```suggestion
    repo = "cfitsio";
    tag = "cfitsio-${finalAttrs.version}";
```
https://github.com/NixOS/nixpkgs/issues/277994